Xamarin.Forms.Core/ResourceDictionary.cs

```diff
@@ -13,7 +14,26 @@ public class ResourceDictionary : IResourceDictionary, IDictionary<string, objec
 		static ConditionalWeakTable<Type, ResourceDictionary> s_instances = new ConditionalWeakTable<Type, ResourceDictionary>();
 		readonly Dictionary<string, object> _innerDictionary = new Dictionary<string, object>();
 
+#pragma warning disable 618
+		[TypeConverter(typeof(TypeTypeConverter))]
+		public Type Source
```

Comment:
@StephaneDelcroix - are you suggesting that I stick with the MergedWith and drop the Source?